rudder-agent/debian/control

```diff
@@ -2,7 +2,7 @@ Source: rudder-agent
 Section: admin
 Priority: extra
 Maintainer: Rudder Team <dev@rudder.io>
-Build-Depends: debhelper (>= 7), bison, gcc, flex, autoconf, automake, libtool, libpcre3-dev, libpam0g-dev, ca-certificates, perl, lsb-release, libyaml-dev, libxml2-dev, libacl1-dev, python, rsync
+Build-Depends: debhelper (>= 7), bison, gcc, flex, autoconf, automake, libtool, libpcre3-dev, libpam0g-dev, ca-certificates, perl, lsb-release, jq (>=9), libyaml-dev, libxml2-dev, libacl1-dev, python, rsync
```

Comment:
you need the -dev here since it is a build dependency

Reply:
anf why the >=9 ?

Reply:
Thought it was debian version.
Fix coming